src/foam/dao/java/JDAO.js

```diff
@@ -27,13 +27,12 @@ foam.CLASS({
 
             // create journal
             setJournal(new foam.dao.FileJournal.Builder(x)
-              .setDao(delegate)
               .setFilename(filename)
               .setCreateFile(true)
               .build());
 
             // create a composite journal of repo journal
-            // and runtime journal and load them all
+// and runti me journal and load them all
```

Comment:
lol "runti me"

Reply:
Should be "runti us" there's no "me" in team.

Reply:
lol i'm not even sure how I changed this line.  Will fix.